client/packages/common/src/ui/components/inputs/Filters/dateFilter.test.tsx

```diff
@@ -0,0 +1,114 @@
+import { getMinMaxDates } from './DateFilter';
+
+describe('getMinMaxDates', () => {
+  it('should return provided min and max dates when no query object', () => {
+    const query = '';
+    const range = 'from';
+    const min = '2025-09-01';
+    const max = '2025-11-01';
```

Comment:
so the min and max in the "DateFilterDefinition" is a seed ?

Reply:
Yeah... well that's the widest range you want to allow. Then as the user inputs the range, we further constrain - you shouldn't allow a "to" that's earlier than the "from"